iib/workers/tasks/build.py

```diff
@@ -115,6 +115,10 @@ def _create_and_push_manifest_list(request_id, arches):
     :rtype: str
     :raises IIBError: if creating or pushing the manifest list fails
     """
+    run_cmd(
+        ['buildah', 'rmi', '--all'],
```

Comment:
It feels unexpected that this method cleans all images known to buildah. Especially because for images, buildah on podman share the same location. Can we just remove the manifest and ignore if it doesn't exist?

Reply:
agreed! based on the description for the method, I expect the behavior to remove only the manifest.

Reply:
Resolved it in #299